index.html

```diff
@@ -45,10 +46,10 @@ <h2 class="new-entry">New Entry</h2>
           </div>
           <div class="column-full">
             <div>
-              <label for="notes">Notes</label>
+              <div class="notes-label"><label for="notes-id">Notes</label></div>
```

Comment:
The `for` attribute should just be `notes`. It needs to match whatever the `id` value is